app/models/grda_warehouse/data_source.rb

```diff
@@ -574,6 +574,33 @@ def hmis_url_for(entity)
     # For any other Enrollment-related record, link to the enrollment page
     url ||= "#{base}/client/#{entity.client&.id}/enrollments/#{entity.enrollment&.id}" if entity.respond_to?(:enrollment) && entity.respond_to?(:client)
 
+    # If we don't have the HMIS driver we probably aren't here, but we need to check for the next section
+    # If we don't have a user, just return the URl (backwards compatibility)
+    return url if user.blank? || ! HmisEnforcement.hmis_enabled?
```

Comment:
This is ok but might be nicer to do
`return unless hmis? && HmisEnforcement.hmis_enabled?` on line 557 instead

Reply:
Will do!  Thanks.